successful-sprint/coding/plan-your-ticket-to-improve-efficency.s.md

```diff
@@ -0,0 +1,51 @@
+# [Standard] Good practices to resolve a ticket
+
+## Owner: Selim BEN AMMAR
+
+## Prerequisites
+
+* I understand well the userStory
+  * check: I know how my PO can validate it
+* After reading the checklist in my ticket, I go through my code to find the files that will be modified.
+* For each file, I find the part of the code that will be targeted.
+* I create a git branch: git checkout -b feature—ticketNumber-ComponentName-FonctionalityShortDescription
+
```

Comment:
Add: 

> * I check the quality of the actual code

The goal is not to start a ticket on a already broken feature.

All transformation processes should be:
* verify the quality of the piece before
* transform the piece
* verify the quality after